pkg/gitproviders/common_test.go

```diff
@@ -213,18 +194,23 @@ func newGitlabTestClient(customTransportFactory gitprovider.ChainableRoundTrippe
 }
 
 func Test_CreatePullRequestToOrgRepo(t *testing.T) {
-
 	accounts := getAccounts()
 
+	githubTestClient, err := newGithubTestClient(SetRecorder(cacheGithubRecorder))
```

Comment:
As this is part of the setup/teardown of the environment for testing, it would be more appropriate to have it in TestMain. From docs `It is sometimes necessary for a test program to do extra setup or teardown before or after testing`. Ref [link](https://golang.org/pkg/testing/#hdr-Main)